Cargo.toml

```diff
@@ -66,7 +66,7 @@ fcm = "0.9"
 ed25519-dalek = "2.0.0-rc.2"
 
 # JWT Authentication
-relay_rpc = { git = "https://github.com/WalletConnect/WalletConnectRust.git", rev = "ced99e7"}
+relay_rpc = { git = "https://github.com/WalletConnect/WalletConnectRust.git", rev = "4ee9007"}
```

Comment:
Must we use hashes on repos we control?
It would be much better to make a release and use the tag in our dependencies.

Reply:
I switched it 👍